test/chainsaw/integration/TTLSecondsAfterFinished-in-jobs/00-assert.yaml

```diff
@@ -9,7 +9,4 @@ metadata:
 spec:
   ttlSecondsAfterFinished: 100
 status:
-  conditions:
-    - type: Complete
-      status: 'True'
```

Comment:
chainsaw has problem to deal with arrays, the `succeeded: 1` is enough of a check that the job succeeded